server/routes.js

```diff
@@ -99,9 +99,19 @@ module.exports = (expressApp, nextApp) => {
       additionalLink = `Additional Link: <a href="${bodyLink}">${bodyLink}</a></br>`;
     }
 
+    let relatedCollectives = 'Related Collectives: ';
+    if (body.relatedCollectives?.length > 0) {
+      for (let i = 0; i < Math.min(body.relatedCollectives.length, 50); i++) {
+        relatedCollectives = `${relatedCollectives}<a href='${body.relatedCollectives[i]}'>${body.relatedCollectives[i]}</a>, `;
```

Comment:
Note that we're introducing yet another HTML injection in the email here

Reply:
@Betree : Do you mean HTML injection through the `relatedCollectives` value? The user doesn't insert any email in this form right? 🤔

Reply:
Attackers can send arbitrary payloads to API endpoints. Not having a text field in the form doesn't mean they can't send the data they want. See https://chrome.google.com/webstore/detail/tamper-chrome-extension/hifhgpdkfodlpnlmlnmhchnkepplebkb